### PR TITLE
[GraphView]Add options for Vertical/horizontal spacing between nodes[gramps51]

### PR DESCRIPTION
**Pull request by vantu5z**
fixes [11155](https://gramps-project.org/bugs/view.php?id=11155)
Снимок экрана от 2019-06-11 11-18-41
Снимок экрана от 2019-06-11 11-18-26

### CODE REVIEW

**Comment by fxtmtrsine** [comment]
Wouldn't that option be better as a button on the tool bar, instead of hidden?

**Comment by vantu5z** [comment]
Like generation spinners? Yes, put it on the toolbar will be better.

**Comment by Mattkmmr** [comment]
Vertical spacing works great, but maybe the family nodes should scale too, so the lines of children rich families aren't packed too tight together. ⬇️

**Comment by fxtmtrsine** [comment]
> Like generation spinners? Yes, put it on the toolbar will be better.

@vantu5z Great 👍

**Comment by vantu5z** [comment]
rebased for Gramps51

**Comment by vantu5z** [comment]
While testing, found that with graphviz 2.42.1 we have problem with scale, it generates cropped SVG.
Have to downgrade to 2.40.1 version.

**Comment by sam-m888** [comment]
Looks good 👍 

Spelling correction
* beetwen  > between

Please change the tool tips to respectively say:

*  "Horizontal spacing between generations"
and 
* "Vertical spacing between generations"

2019-09-19 10_03_56-Window

**Comment by sam-m888** [comment]
The popovers are much more compact 👍 

generations to show

chart spacings

File: GraphView/graphview.py [hunk 1]
```diff
@@ -899,6 +924,16 @@ def __init__(self, view, dbstate, uistate):
         # used for popup menu, prevent destroy menu as local variable
         self.menu = None
 
+    def spinners_popup(self, widget, popover):
+        """
+        Popover for generations and spacing params.
+        Different popup depending on gtk version.
```
Comment (sam-m888) [inline GraphView/graphview.py]: 👍

**Comment by sam-m888** [comment]
Thank you

**Pull request merged**